FirebaseAuth/Sources/Backend/RPC/Proto/TOTP/FIRAuthProtoFinalizeMFATotpSignInRequestInfo.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "FirebaseAuth/Sources/Backend/RPC/Proto/FIRAuthProto.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface FIRAuthProtoFinalizeMFATotpSignInRequestInfo : NSObject <FIRAuthProto>
+
+@property(nonatomic, strong, readonly, nullable) NSString *mfaEnrollmentID;
+
+@property(nonatomic, strong, readonly, nullable) NSString *verificationCode;
+
+- (instancetype)initWithVerificationCode:(NSString *)verificationCode
```

Comment:
Nit: it seems more intuitive to me to put mfaEnrollmentID as the first parameter

Reply:
Updated